draft-ietf-quic-transport.md

```diff
@@ -187,8 +187,9 @@ QUIC:
 
 QUIC packet:
 
-: The smallest unit of QUIC that can be encapsulated in a UDP datagram. Multiple
-  QUIC packets can be encapsulated in a single UDP datagram.
+: A complete processable payload with a packet type, encryption level and
+  packet number. Multiple QUIC packets can be encapsulated in a single UDP
+  datagram.
 
```

Comment:
An authenticated and encrypted container of payload usually carrying a packet number and sometimes a connection identifier and version information in its header.